source/_components/ifttt.markdown

```diff
@@ -13,7 +13,35 @@ featured: true
 ha_iot_class: "Cloud Push"
 ---
 
-[IFTTT](https://ifttt.com) is a web service that allows users to create chains of simple conditional statements, so called "Applets". With the IFTTT component you can trigger applets through the **"Webhooks"** service (which was previously the **"Maker"** channel). See the [announcement blog post](/blog/2015/09/13/home-assistant-meets-ifttt/) for examples how to use it.
+[IFTTT](https://ifttt.com) is a web service that allows users to create chains of simple conditional statements, so-called "Applets". With the IFTTT component, you can trigger applets through the **"Webhooks"** service (which was previously the **"Maker"** channel).
+
+## {% linkable_title Sending events from IFTTT to Home Assistant %}
+
+To be able to receive events from IFTTT, your Home Assistant instance needs to be accessible from the web ([Hass.io instructions](/addons/duckdns/)) and you need to have the `base_url` configured for the HTTP component ([docs](https://www.home-assistant.io/components/http/#base_url)).
+
+To set it up, go to the integrations page in the configuration screen and find IFTTT. Click no configure. Follow the instructions on the screen to configure IFTTT.
+
+Events coming in from IFTTT will be available as events in Home Assistant and are fired as `ifttt_webhook_received`. The data specified in IFTTT will be available as the event data. You can use this event to trigger automations.
+
+For example, set the body of the IFTTT webhook to:
+
+```json
+{ "service": "light.turn_on", "entity_id": "light.living_room" }
+```
+
+You can then consume that information with the following automation:
+
+```yaml
+automation:
+  trigger:
+    event: ifttt_webhook_received
```

Comment:
How can I trigger an automation base on the content of event_data?
```yaml
trigger:
  event: ifttt_webhook_received
  event_data:
    action: call_service
```

Also change the above json payload to
```json
{ "action": "call_service", "service": "light.turn_on", "entity_id": "light.living_room" }
```